### PR TITLE
Refine task list UX and task editing workflow

### PR DESCRIPTION
**Pull request by m4thayus**
Makes many changes but a few highlights are:
* Adding status, points, and point estimate columns to the task table
* Improves styling of the task edit/new form (w/ new fields)
* Simplifies the task show page
* Improves parse-ability of task list with sorting and breaking into different tables
* Adds an xlarge size and uses css and color to indicate sizes in general

### CODE REVIEW

File: app/models/task.rb [hunk 1]
```diff
@@ -11,10 +13,23 @@ class Task < ApplicationRecord
   validates :title, presence: true
   validates :size, inclusion: { in: SIZES }, allow_nil: true
 
-  scope :active, -> { where(completed_at: nil) }
+  scope :available, -> { where(started_at: nil, completed_at: nil) }
+  scope :active, -> { where.not(started_at: nil).where(completed_at: nil) }
   scope :recently_completed, -> { where("completed_at > ?", 1.month.ago) }
   scope :approved, -> { where(approved: true) }
 
+  def <=>(other)
```
Comment (zoeblairfriedman) [inline app/models/task.rb:21]: Cool mixin!

File: app/views/tasks/_table.html.slim [hunk 2]
```diff
@@ -2,55 +2,51 @@ table
   caption
     = caption
   thead
-    - %w[task title size point_of_contact started_on expected_on expected_duration].each do |header|
+    th &check;
```
Comment (zoeblairfriedman) [inline app/views/tasks/_table.html.slim:5]: What is this syntax? I can't figure it out.

File: spec/features/task_list_spec.rb [hunk 3]
```diff
@@ -28,7 +28,7 @@
     end
 
     it "shows a column for reviewed tasks" do
```
Comment (zoeblairfriedman) [inline spec/features/task_list_spec.rb:30]: I'm not sure why the reviewed test was still there—but that's not really a thing anymore, is it? We only have "approved" now, yes?

File: spec/features/task_list_spec.rb [hunk 4]
```diff
@@ -28,7 +28,7 @@
...
-      expect(page).to have_text "Reviewed"
+      expect(page).to have_text "&check;"
```
Comment (zoeblairfriedman) [inline spec/features/task_list_spec.rb:31]: I'm not sure why the reviewed test was still there—but that's not really a thing anymore, is it? We only have "approved" now, yes?

File: app/views/tasks/_table.html.slim (hunk 2 above)
Comment (m4thayus) [reply app/views/tasks/_table.html.slim:5]: It's an html entity and some of them have nice shorthands
https://www.compart.com/en/unicode/U+2713

File: spec/features/task_list_spec.rb (hunk 4 above)
Comment (zoeblairfriedman) [reply spec/features/task_list_spec.rb:31]: Totally understandable.

File: app/views/tasks/_table.html.slim (hunk 2 above)
Comment (zoeblairfriedman) [reply app/views/tasks/_table.html.slim:5]: ahhhh cool cool. Thanks!

**Pull request merged**